content/en/contribute/code/releasing/_index.md

```diff
@@ -44,6 +44,17 @@ Given a version number `MAJOR.MINOR.PATCH`, increment the:
 {{% alert title="Info" %}} You can find the versions currently supported, dependencies, and release notes for the CHT Core 
  [on the Releases page]({{% ref "core/releases" %}}). {{% /alert %}}
 
+### When to wait to release
+
+In general, we do not want to wait to do a release.  The moment the release manager feels the release is ready, they should release it.  This may even be with just one ticket! 
```

Comment:
[Per the guidelines](https://docs.communityhealthtoolkit.org/contribute/docs/style-guide/#using-we), using `we` can be confusing for the reader. There are several mentions of it in this new section and I suggest we rephrase it to avoid the `we`/`us` confusion. 

```suggestion
In general, release managers should not feel they have to wait to do a release. The moment the release manager feels the release is ready, they should release it. This may even be with just one ticket! 
```